src/validation/rules/ExecutableDefinitions.js

```diff
@@ -28,10 +29,7 @@ export function ExecutableDefinitions(
   return {
     Document(node) {
       for (const definition of node.definitions) {
-        if (
-          definition.kind !== Kind.OPERATION_DEFINITION &&
-          definition.kind !== Kind.FRAGMENT_DEFINITION
-        ) {
+        if (!isExecutableDefinitionNode(node)) {
```

Comment:
The flow error here is because you're asking if the Document is an ExecutableDefinitionNode rather than asking if the definition is.

Reply:
@mjmahone Fixed. Thanks.